lib/electric_slide/call_queue.rb

```diff
@@ -276,11 +276,9 @@ def bridge_agent(agent, queued_call)
         remove_agent agent
       end
 
-      agent.call.on_joined do 
-        agent.callback :connect, self, agent.call, queued_call
-      end
-
       agent.join queued_call
+      agent.callback :connect, self, agent.call, queued_call
```

Comment:
So there was a benefit to calling this in the Call's join callback, which is that this would be executed in that Call actor's thread, rather than blocking the queue itself. Any arguments in favour of this change?